### PR TITLE
Don't pin to latest tag

### PR DESCRIPTION
**Pull request by damccorm**
Its usually better to suggest a custom tag to avoid overwriting containers from old jobs

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://github.com/apache/beam/blob/master/CONTRIBUTING.md#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI or the [workflows README](https://github.com/apache/beam/blob/master/.github/workflows/README.md) to see a list of phrases to trigger workflows.

### CODE REVIEW

**Comment by damccorm** [comment]
R: @tvalentyn

File: examples/notebooks/beam-ml/run_inference_vllm.ipynb [hunk 1]
```diff
@@ -352,11 +352,11 @@
         "\n",
         "1.   In the sidebar, click **Files** to open the **Files** pane.\n",
         "2.   In an environment with Docker installed, download the file **VllmDockerfile** file to an empty folder.\n",
-        "3. Run the following commands. Replace `<REPOSITORY_NAME>` with a valid [Artifact Registry](https://cloud.google.com/artifact-registry/docs/overview) repository.\n",
+        "3. Run the following commands. Replace `<REPOSITORY_NAME>:<TAG>` with a valid [Artifact Registry](https://cloud.google.com/artifact-registry/docs/overview) repository and tag.\n",
```
Comment (tvalentyn) [inline examples/notebooks/beam-ml/run_inference_vllm.ipynb:355]:
```suggestion
        "3.   Run the following commands. Replace `<REPOSITORY_NAME>:<TAG>` with a valid [Artifact Registry](https://cloud.google.com/artifact-registry/docs/overview) repository and tag.\n",
```

File: examples/notebooks/beam-ml/run_inference_vllm.ipynb [hunk 2]
```diff
@@ -396,7 +396,7 @@
         "options = PipelineOptions()\n",
         "\n",
         "BUCKET_NAME = '<BUCKET_NAME>' # Replace with your bucket name.\n",
-        "CONTAINER_LOCATION = '<REPOSITORY_NAME>' # Replace with your container location (<your_gar_repository> from the previous step)\n",
+        "CONTAINER_LOCATION = '<REPOSITORY_NAME>' # Replace with your container location (<REPOSITORY_NAME>:<TAG> from the previous step)\n",
```
Comment (tvalentyn) [inline examples/notebooks/beam-ml/run_inference_vllm.ipynb:399]:
```suggestion
        "CONTAINER_IMAGE = '<REPOSITORY_NAME>:<TAG>'  # Replace with the image repository and tag from the previous step.\n",
```

**Review by tvalentyn — APPROVED** [review]
thanks!

File: examples/notebooks/beam-ml/run_inference_vllm.ipynb [hunk 3]
```diff
@@ -373,7 +373,7 @@
         "First, define the pipeline options that you want to use to launch the Dataflow job. Before running the next cell, replace the following variables:\n",
         "\n",
         "- `<BUCKET_NAME>`: the name of a valid [Google Cloud Storage](https://cloud.google.com/storage?e=48754805&hl=en) bucket. Don't include a `gs://` prefix or trailing slashes.\n",
-        "- `<REPOSITORY_NAME>`: the name of the Google Artifact Registry repository that you used in the previous step. Don't include the `latest` tag, because this tag is automatically appended as part of the cell.\n",
+        "- `<REPOSITORY_NAME>`: the name of the Google Artifact Registry repository that you used in the previous step. \n",
```
[the remark this answers is not in the document]
Comment (damccorm) [reply examples/notebooks/beam-ml/run_inference_vllm.ipynb:376]: Thanks, missed this!

**Pull request merged**